dev/Repeater/ItemsRepeater.idl

```diff
@@ -543,6 +543,7 @@ runtimeclass IndexPath : Windows.Foundation.IStringable
 [default_interface]
 runtimeclass SelectionModelSelectionChangedEventArgs
 {
+    Windows.Foundation.Collections.IVectorView<IndexPath> PreviousSelectedIndices{ get; };
```

Comment:
Thanks!

Reply:
Also, It probably makes sense to give the currently Selected Indices. Please file a bug for that.

Reply:
[https://github.com/microsoft/microsoft-ui-xaml/issues/1645](https://github.com/microsoft/microsoft-ui-xaml/issues/1645)

---
In reply to: [347421487](https://github.com/microsoft/microsoft-ui-xaml/pull/1623#discussion_r347421487) [](ancestors = 347421487)